KPI-team1-frontend/src/pages/KpiPage.tsx

```diff
@@ -26,23 +26,38 @@ const HEADER_KPI_COLUMNS: GridColDef[] = [
     sortable: true,
     hideable: false,
     headerAlign: "center",
-  },
-  {
-    headerName: "Target",
-    field: "kpi_target",
-    width: 150,
-    sortable: false,
-    headerAlign: "center",
-    align: "center",
+    renderCell: (params) => {
+      if (params.row.unit === "%") {
+        return (
+          <>
+            <span>{params.value}</span>
+            <div className="ml-4 w-6 h-6 bg-amber-400 rounded-[100px] flex-col justify-center items-center gap-2 inline-flex">
+              <div className="w-[19px] text-center text-zinc-700 text-base font-medium font-['Inter']">
+                %
+              </div>
+            </div>
+          </>
+        );
+      }
+    },
   },
   {
     headerName: "Latest Value",
     field: "latest_value",
     width: 150,
     sortable: true,
     headerAlign: "center",
-
     align: "center",
+    renderCell: (params) => {
+      const value = params.value as number;
+      if (value === null) {
+        return null;
+      } else if (params.row.unit === "%") {
+        return value.toFixed(2) + "%";
+      } else {
+        return value.toLocaleString("fr-ch");
```

Comment:
Translations would be nice but for now I think we need to assume English is the default

Reply:
but it was mostly to have the swiss seperators : 1 000 000 
![image](https://github.com/WomenPlusPlus/deploy-impact-23-kpi-1/assets/42074299/43f92267-cebe-421c-862c-da7ad1e1f704)

with the English one, we have that , as i dont think "en-ch" works because i get that 1'000'000 :
![image](https://github.com/WomenPlusPlus/deploy-impact-23-kpi-1/assets/42074299/bc3dfd74-0c2e-41a6-9b42-a6c25f368a57)

Reply:
How about en-us or en-gb?